Snowflake.Data/Client/SnowflakeObjectConstructionMethod.cs

```diff
@@ -0,0 +1,9 @@
+namespace Snowflake.Data.Client
+{
+    public enum SnowflakeObjectConstructionMethod
```

Comment:
why not StructuredObjectConstructionMethod and less SnowflakeObject~?

Reply:
didn't we want to have all public classes to have Snowflake prefix?